### ISSUE TITLE
Display generic error message on Login Page when email/password is wrong

### ISSUE THREAD

**Issue by C-Anirudh** [C-Anirudh/chuck#4]
Currently the errors in the login page are very specific, i.e. ,  the alert `Invalid Email address` is displayed when the email address is wrong and the alert `Invalid Password Provided` is displayed when the password is wrong. 

This is not a good security practice and the error alert should be changed to something generic like `Invalid email address or password` , which is displayed when either is wrong.

You will have to create a new generic error in `service/models/users.go`  and modify the `Authenticate` function to display it.

**Comment by cdbm** [comment]
i'll work on this

**Comment by cdbm** [comment]
hey @C-Anirudh i'm a little stuck here, when i try to signup or login i get this on console.

`POST http://0.0.0.0:9000/signup net::ERR_ADDRESS_INVALID `

i tried to change the  port number and got same error, do you have any ideia of whats causing this?

**Comment by C-Anirudh** [comment]
@cdbm I haven't faced this error yet, did this error pop up from the original code or after you made some changes ?

**Comment by cdbm** [comment]
i just ran the scripts and setted up the database, but instead of run `go get github.com/C-Anirudh/chuck` i forked the repository and cloned via github, it could be it?

**Comment by C-Anirudh** [comment]
@cdbm No, there should be no problem even if you clone it instead of using `go get` . Note that here `http://0.0.0.0:9000` is where the go server is running, the port has been defined in `service/main.go`.

**Comment by cdbm** [comment]
yeah, the ports are ok at the code, i'll look more deeply soon.

**Comment by C-Anirudh** [comment]
I can't think of a reason for the error at the moment, I will let you know if I come up with something, you can join the gitter channel (link in readme) to discuss errors, etc
